app/src/app/list/list.module.ts

```diff
@@ -28,6 +28,8 @@ import { ListComponent } from './list.component';
 import { PollsTableComponent } from './polls-table/polls-table.component';
 import { DeletePollDialog } from './list.service';
 
+import {MatTabsModule} from '@angular/material/tabs';
```

Comment:
Please move that line in the block of imports from Angular Material, line 22. Please also add spaces to match how it is done on the other lines.